### DISCUSSION TITLE
How to record multiple domains

### DISCUSSION THREAD

**Discussion by rookybird** [eProsima/DDS-Record-Replay#103] [Q&A]
Hi all, 
I am wondering how to use Fast DDS recorder to record multiple domains at the same time (data from different domains is stored in the same mcap file)

**Comment by juanlofer-eprosima** [comment]
Hi @rookybird ,

This feature is currently not available, you would need to launch a DDS-Recorder instance for each domain and merge the resulting mcap files as a postprocessing step.

Feel free to [contact eProsima's commercial support team](https://github.com/eProsima/Fast-DDS#getting-help) to discuss options on how to add this functionality to the product.
